maker/src/main.rs

```diff
@@ -191,11 +191,13 @@ async fn main() -> Result<()> {
     let auth_username = rocket_basicauth::Username("itchysats");
     let auth_password = seed.derive_auth_password::<rocket_basicauth::Password>();
 
-    let (identity_pk, identity_sk) = seed.derive_identity();
+    let keypair = seed.derive_identities();
```

Comment:
❓Why was this renamed to `keypair`? I feel identities still expresses better what this is.

Reply:
It seems like destructuring would make sense here (but yes, "keypair" isn't quite right).

Reply:
good catch! was meant to be renamed to identities indeed. Unfortunately, can't destructure here as it's being passed to MakerActorSystem